src/tests/LogIn.test.js

```diff
@@ -0,0 +1,52 @@
+import { render, screen, fireEvent } from '@testing-library/react';
+import { GlobalContext } from '../context/GlobalContext';
+import { BrowserRouter } from 'react-router-dom';
+import LogIn from '../pages/LogIn';
+
+describe('LogIn', () => {
+  test('renders the LogIn component', () => {
+    const initialState = {};
+    const { getByRole, getByPlaceholderText } = render(
+      <GlobalContext.Provider value={initialState}>
+        <BrowserRouter>
+          <LogIn />
+        </BrowserRouter>
+      </GlobalContext.Provider>
+    );
+    const title = getByRole('heading');
+    expect(title).toBeInTheDocument();
+    expect(title.textContent).toBe('Log in');
```

Comment:
I think you should divide this. A test just to check for example the title to be login and a completely different one for really testing the log in